tools/cell_census_builder/tests/anndata/test_anndata.py

```diff
@@ -0,0 +1,62 @@
+from typing import List
```

Comment:
any reason to not include a simple test for the schema version?  (ie., that we drop any unsupported schema versions)?

Reply:
I would also add a test to ensure it correctly handles the case where the AnnData lacks a raw (i.e., adata.raw is None)

Reply:
Good idea, I will add both.